lib/main_common.dart

```diff
@@ -88,8 +91,15 @@ Future<void> _firebaseInitializer(FlavorName flavorName) async {
 }
 
 void _activateManagers() {
+  activateConnectionManager();
   activateRemoteConfig();
```

Comment:
Add a connection check to activate the manager. 
Add the condition in the "remote_config_manager" file before the initialisation